tests/cover/test_testdecorators.py

```diff
@@ -216,9 +216,9 @@ def test_removing_an_element_from_a_unique_list(xs, y):
 
 
 @fails
-@given(lists(integers(), average_size=25.0), integers())
-def test_removing_an_element_from_a_non_unique_list(xs, y):
-    assume(y in xs)
+@given(lists(integers(), average_size=25.0), data())
+def test_removing_an_element_from_a_non_unique_list(xs, data):
+    y = data.draw(sampled_from(xs))
```

Comment:
I think the test could actually fail because `xs == []`, rather than anything about uniqueness.

We could also simplify the test - instead of drawing an integer sampled form the list and removing it, just `xs.pop()` because order isn't important here.

Reply:
> I think the test could actually fail because xs == [], rather than anything about uniqueness.

No, that's fine.  `sampled_from([])` is just an empty strategy and drawing from it is equivalent to `reject()`.